builder/azure/common/client/azure_authorizer.go

```diff
@@ -79,6 +81,15 @@ func buildAuthorizer(ctx context.Context, authOpts AzureAuthOptions, env environ
 			TenantID:                      authOpts.TenantID,
 			OIDCAssertionToken:            authOpts.ClientJWT,
 		}
+	case AuthTypeOidcURL:
+		authConfig = auth.Credentials{
+			Environment:                         env,
+			EnableAuthenticationUsingGitHubOIDC: true,
+			ClientID:                            authOpts.ClientID,
+			TenantID:                            authOpts.TenantID,
+			GitHubOIDCTokenRequestURL:           authOpts.OidcRequestUrl,
```

Comment:
Is that your choice to call that `GitHubOIDC`? This feels unnecessarily tied to Github where the workflow of a token/URL to refresh the token from is hardly a specificity of the platform

Reply:
That is the SDK field name, and not a name I chose.  That authorizer is available here https://github.com/hashicorp/go-azure-sdk/blob/main/sdk/auth/github_oidc_authorizer.go and is the one we have exposed to us,